### ISSUE TITLE
Timberwolves

### ISSUE THREAD

**Issue by Lussian-A** [Filigrani/SkyCoop#27]
**Mod Version:** v0.10.3
**Game Version:** v20.2 
**Launcher:** Steam 
**Additional mods installed:** No 
**Perspective:**  Both

### Description
Timberwolf spawn is well bugged in a way. As it currently stands BL and BR are super easy due to low to non spawning of normal wolves. As it's mainly suppose to be a timberwolf spawn area. But timberwolves only spawn one and one and not as a pack and thus always wimper and run away from you, meaning you can walk around without any danger what so ever in these "hard" but lucrative areas. 

### How to reproduce
Start multiplayer spawn in BL or BR and walk around until you meet a single timberwolf that will run away from you.

### Expected behavior
Expected packs of timberwolves

Possibly solution until you've found a way to fix timberwolves. Replace spawn with regular wolves so there's somewhat of a threat there. 

Sorry for the constant bug reports :P

If non-critical, just close and pick it up when time allows. I mean you are turning a singleplayer game into a multiplayer one, there's bound to be weird bugs popping up here and there. 

Thanks again.

**Comment by Filigrani** [comment]
They spawns alones probably because of how my recreation of spawning code works. I will check this out. But honestly, even with correct amout of them there still problems of them's "moral metter", because every time wolf re-target to other player, they become an "inactive dummy" that controlled by other player, and if not  all of them on one player they will lose "moral meter units", this has not been fixed, because there no good solutions of fixing this in my mind. So best solution of this for current situation, will be auto-replace spawns of that wolfs with regular ones, but just set correct amount of them, but I not sure if it will be harder, cause they still will flee from one shot. By the way, give me your steam ID, I will give you special flair in game for good reports.

**Comment by Lussian-A** [comment]
Yeah I figured the moral meter was gonna be a hard nut to crack. But since a single timberwolf is programmed to flee whenever it sees the player, regular wolves will at least try to mess with ya and it makes the place at least not a complete walk over. :D

https://steamcommunity.com/id/lussian/ Not sure if that's what you need or the 64 bit number string. Or just current username which is Lussian

Thanks and keep up the good work.

**Comment by Filigrani** [comment]
Ye, I was need 64 one, I found it, and added flair, so now you got pin with crossed bug.
